Directory.Build.targets

```diff
@@ -1,4 +1,9 @@
 <Project>
+  <!-- Explicitly add framework reference -->
+  <ItemGroup Condition=" @(FrameworkReference) == '' And ($(TargetFramework.StartsWith('net5'))) ">
+    <FrameworkReference Include="Microsoft.NETCore.App" />
```

Comment:
Will you help me understand what this ItemGroup is doing specifically to the build so I don't screw it up with future changes?

Reply:
Sure.
It is a special automagical reference .

See the documentation https://github.com/NuGet/Home/wiki/%5BSpec%5D-FrameworkReference-in-NuGet

it is referenced implicitly unless you specify DisableImplicit as I did.